mifospay/src/main/res/values/strings.xml

```diff
@@ -235,4 +238,9 @@
     <string name="amount">Amount</string>
     <string name="merchant_history_message">The transaction history with this merchant will be shown here</string>
     <string name="log_out_title">Do you want to logout?</string>
+    <string name="test_user">Naman Dwivedi 2</string>
```

Comment:
Add some generic name here instead of `Naman Dwivedi 2`.

Reply:
okay